docs/user-guide/ze-install.md

```diff
@@ -3,46 +3,47 @@
+The Zowe Explorer extension for Visual Studio Code (VS Code) modernizes the way developers and system administrators interact with z/OS mainframes, and lets you interact with data sets, USS files, and jobs.
 
-* Enables you to create, modify, rename, copy, and upload data sets directly to a z/OS mainframe.
-* Enables you to create, modify, rename, and upload USS files directly to a z/OS mainframe.
-* Provides a more streamlined way to access data sets, USS files and jobs.
-* Lets you create, edit, and delete Zowe CLI `zosmf` compatible profiles.
-* Lets you use the Secure Credential Store plug-in to store your credentials securely in the settings.
+Install the extension directly to [VSCode](https://code.visualstudio.com/) to enable the extension within the GUI. Working with data sets and USS files from VSCode can be more convenient than using 3270 emulators, and complements your Zowe CLI experience. The extension provides the following benefits:
+
+- Enables you to create, modify, rename, copy, and upload data sets directly to a z/OS mainframe.
+- Enables you to create, modify, rename, and upload USS files directly to a z/OS mainframe.
+- Provides a more streamlined way to access data sets, USS files, and jobs.
+- Lets you create, edit, and delete Zowe CLI `zosmf` compatible profiles.
 
 **Note:** Zowe Explorer is a subcomponent of [Zowe](https://zowe.org/home/). The extension demonstrates the potential for plug-ins powered by Zowe.
 
 ## Software Requirements
 
 Ensure that you meet the following prerequisites before you use the extension:
 
-* Get access to z/OSMF.
-* Install [Node.js](https://nodejs.org/en/download/) v8.0 or later.
-* Install [VSCode](https://code.visualstudio.com/).
-* Configure TSO/E address space services, z/OS data set, file REST interface, and z/OS jobs REST interface. For more information, see [z/OS Requirements](https://docs.zowe.org/stable/user-guide/systemrequirements-zosmf#z-os-requirements).
-* Create one Zowe CLI `zosmf` profile so that the extension can communicate with the mainframe.
+- Get access to z/OSMF.
+- Install [Node.js](https://nodejs.org/en/download/) v8.0 or later.
+- Install [Visual Studio Code](https://code.visualstudio.com/).
+- Configure TSO/E address space services, z/OS data set, file REST interface, and z/OS jobs REST interface. For more information, see [z/OS Requirements](https://docs.zowe.org/stable/user-guide/systemrequirements-zosmf#z-os-requirements).
+- Create a Zowe CLI `zosmf` profile so that the extension can communicate with the mainframe.
 
 ### Profile notes:
 
-   - You can use your existing Zowe CLI `zosmf` profiles that are created with the Zowe CLI v.2.0.0 or later.
+- Use existing Zowe CLI `zosmf` profiles created with Zowe CLI v.2.0.0 or later.
```

Comment:
Personally, I will keep the suggestive wording, as we don't want to give people the idea that they have to use profiles created from the CLI

Reply:
Updated per your comment